src/components/Details/VmDetails/fixtures/VmDetails.fixture.js

```diff
@@ -0,0 +1,225 @@
+import { VmDetails } from '../VmDetails';
+import { PodModel } from '../../../../models';
+
+const vm = {
```

Comment:
can you move this to https://github.com/kubevirt/web-ui-components/tree/master/src/k8s under mock_vm ?

Reply:
Done!